aws/ecs/lib/resource/database.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ Copyright (c) 2016-2020, CKSource - Frederico Knabben. All rights reserved.
+ */
+
+import { Construct, RemovalPolicy } from '@aws-cdk/core';
+import {
+	AuroraMysqlEngineVersion,
+	DatabaseCluster,
+	DatabaseClusterEngine,
+	Credentials
+} from '@aws-cdk/aws-rds';
+import {
+	InstanceClass,
+	InstanceSize,
+	InstanceType,
+	SubnetType
+} from '@aws-cdk/aws-ec2';
+import { ISecret, Secret } from '@aws-cdk/aws-secretsmanager';
+
+import { Network } from './network';
+
+interface IDatabaseProps {
+  network: Network;
+}
+
+export class Database extends Construct {
```

Comment:
I would add a piece of information, about what each file does, ex:   
  
// The script prepares databases infrastructure.   
// It will store all the persistent data from the CKEditor like comments, suggestions etc.  
// ....  
export class Database extends Construct {